cs6232-g4/Controller/FurnitureController.cs

```diff
@@ -7,7 +7,7 @@
 using Employees.DAL;
 using Furnitures.Model;
 
-namespace Members.Controller
+namespace Furnitures.Controller
```

Comment:
General comments:

1- the program crash and doesn't run
2- the search furniture is its own form, but **it should be a user control.** which gets imported into the dashboard/main form 
3- no DAL file. So how do we retrieve the data from the database?